go/src/socialapi/workers/oauth/api/api.go

```diff
@@ -1,100 +0,0 @@
-package main
```

Comment:
Glide complained about it when reading imports / updating deps. Instead of fixing build / deps I just removed it as it was not used.

Do you want it back?